Enforcement landing page

adborden opened the pull request:
Part of https://github.com/18F/fec-eregs/issues/224

Contains draft content from https://github.com/18F/fec-eregs/issues/233

Removes the filter panel on legal landing pages from https://github.com/18F/fec-eregs/issues/242
- [x] Depends on https://github.com/18F/fec-style/pull/523

adborden committed:
Route and template for EM landing page

adborden committed:
EM landing with draft content

adborden commented:
cc @jenniferthibault

jenniferthibault commented:
Looking here: http://127.0.0.1:3000/legal/enforcement-matters/

(Ignore the floating feedback button, that's an artifact of my screen cap tool!)

![screencapture-127-0-0-1-3000-legal-enforcement-matters-1475012039070](https://cloud.githubusercontent.com/assets/11636908/18893813/7910e672-84dd-11e6-8286-fb38511776d7.png)




There is so much "coming soon" to this page that I'm hesitant to offer design improvements yet. 

If it were shipping as is, the big issue to my 👀 is really awkward spacing, which I have some suggested fixes for. Since those are contingent to the exact copy of the page, I'd rather hold on them until final copy is in place, then resolve them. Hoping to not make design fixes twice.

Is that ok with you @adborden ?

adborden commented:
Yep, I agree. We'll do a more thorough design review when I update the content. The search bar is looking okay, though?

Commit:
Adds example text class

This has the correct top margin, but a funky customized left margin
that’s not quite right

Commit:
Use large search pattern

jenniferthibault commented:
I do have a few thoughts for the search bar, but they might fall under `fec-style` 😺 

I think `search--mini` should probably be used in cases where it's in a narrow column, like the filter panel. It feels proportionally wrong in a wide space like this. I nudged it back to `search--large` which feels _more_ proportionally correct, but I think I was looking for something with an in-between height, which we don't have. Is it worth making a `search--medium` ?

The other thing is the Example text below the search bar has an existing class (that I added) that captures the correct margin-top, but applies a less-useful margin-left. It seems like the margin-left is a specific use case for the `search--large` class when there is also a dropdown, like on the data landing page: https://beta.fec.gov/data/

But more often, we want to use that class without the margin-left. It seems like we should refactor that pattern to have no margin left as the base unit, then add a class/something to give it the left space on the data landing page instance. 

Here's the screenshot of both of those applied, but I know we'll probably want to retweak.

<img width="1016" alt="screen shot 2016-09-28 at 9 59 46 am" src="https://cloud.githubusercontent.com/assets/11636908/18916384/4012d4e0-8562-11e6-94f1-530bbe436a51.png">


Am I out of scope/talking crazy? Let me know what you think

adborden commented:
Yep, makes sense! I'll follow up with the changes. Should the search bar be less than full-width like in your screenshot? Trying to stick the uswds grid, so if you have a specific width in mind, I'll use that.

jenniferthibault commented:
Nope, I don't have a specific width in mind, and yep, it's ok for it to not be full-width. 

For height on the medium sized search bar, could we try 4.5rem? This would also be the pattern we'd use for search bars on the other legal landing page templates that we talked about adjusting in https://github.com/18F/fec-eregs/issues/242. 

cc @noahmanger into this for visibility into pattern tweaks 👀

adborden committed:
Update enforcement urls

adborden committed:
Special styling for border-radius on search combo

adborden committed:
Add class for search combo with select

emileighoutlaw committed:
Adds OGC approved content

adborden committed:
Use new heading styles

adborden commented:
@noahmanger updated this with the new heading styles.

adborden commented:
@jenniferthibault here is the screen capture.
![screencapture-localhost-3000-legal-enforcement-1475776041317](https://cloud.githubusercontent.com/assets/509703/19163842/6e14d1f8-8bb2-11e6-8f24-57da41ad2f00.png)

adborden committed:
Fix spacing after the MURs inline search

adborden commented:
And fixed the spacing between the MURs search and the "Administrative Fine Program" header.

emileighoutlaw committed:
fixes typo

emileighoutlaw commented on openfecwebapp/templates/legal-enforcement-landing.html:
```diff
@@ -0,0 +1,95 @@
+</header>
+<div class="content__section">
+  <p>
+    The FEC has exclusive jurisdiction over the civil enforcement the federal campaign finance law. 
```
You're definitely right. I made that change, and good catch @adborden

emileighoutlaw commented:
That coming soon text for audits does look really awkward. 

Perhaps I should send a screenshot to Amy and Cheryl and ask if we can cut it down to something minimal like:

> "Coming soon: Commission audits of political committees"

??

Commit:
Adds p class to list items

This makes sure they fall within the container, and matches our
approach on the Statutes landing page. cc @adborden

Commit:
Adding italic classes for coming soon

This clarifies current and coming content

noahmanger reviewed (approved):
This looks good to me. Once @emileighoutlaw 's content point is resolved feel free to merge.

Commit:
Moving text to search label

Design change based on copy updates

noahmanger commented:
Oops, recanting my approval for the time being while Jen finishes some things up.

jenniferthibault commented:
Sorry Noah, that's my bad. I was experimenting on my own without communicating. 

I did a few small things that will improve the design regardless of the design changes that will come from adding the legal citations the general counsel wants.

- first, I added a `<p>` class to the list items, like we did on the statutes landing page, to make sure they stay within the container.
<img width="1063" alt="screen shot 2016-10-06 at 8 06 38 pm" src="https://cloud.githubusercontent.com/assets/11636908/19174915/6801c5fc-8c00-11e6-88cf-4b2f2e9ca6b1.png">

- Then, I made all "coming soon" text italic to set it off from the more settled content. Since there's so much of it in this layout, I found it helpful.
- Last, I moved the "Search or browse closed MURs" into the search container, where it felt closer to the action and more relevant. This is the only place I took a content liberty @emileighoutlaw and removed the word "current" because the search also goes across archived MURs. 

<img width="1053" alt="screen shot 2016-10-06 at 8 06 30 pm" src="https://cloud.githubusercontent.com/assets/11636908/19174919/6b400daa-8c00-11e6-9470-183030b21ee4.png">


I don't feel like this design is restful on the eyes overall right now, but it's ok enough to wrap this PR and look at what the new citations section will do to the layout, then get picky. 👌 

Thanks for your patience everyone, I know this PR and this page have been a bit all over the place.

noahmanger commented on openfecwebapp/templates/legal-enforcement-landing.html:
```diff
@@ -62,15 +62,13 @@
           <div class="row">
             <div class="usa-width-one-whole">
               <form action="{{ url_for('murs') }}" autocomplete="off" class="search__container  js-search">
+                <label class="label">Search or browse MURs</label>
```
This `<label>` needs `for="search-input"` so it's associated with the input explicitly.

noahmanger commented on openfecwebapp/templates/legal-enforcement-landing.html:
```diff
@@ -0,0 +1,103 @@
+<header class="heading--main">
+  <h1>Enforcement matters</h1>
+</header>
+<div class="content__section">
```
Instead of nesting `<p>` tags in `<li>` tags, let's just add `content__section--narrow` to this `div` like we do below. It will achieve the same effect.

noahmanger reviewed (changes requested):
Ok, commented on just a couple things @jenniferthibault . I can also make the changes too.

Commit:
Using content__section--narrow and adding label for attr

noahmanger commented:
After checking with @jenniferthibault I'm going to merge this now and content edits can follow.

jenniferthibault commented:
Picking up where this leaves off over in https://github.com/18F/openFEC-web-app/pull/1622
